test/integration/proposals_config.ts

```diff
@@ -19,6 +20,23 @@ const proposals: ProposalsConfigMap = {
     category: ProposalCategory.None,
     totalValue: 0,
     proposal: undefined
+  },
+  fip_62: {
+    deploy: true,
+    proposalId: undefined,
+    affectedContractSignoff: [
+      'ethPSM',
+      'PSMRouter',
+      'aaveEthPCVDripController',
+      'collateralizationOracle',
+      'bondingCurve',
+      'ethReserveStabilizer',
+      'ethReserveStabilizerWrapper'
```

Comment:
withdraw all ETH using RatioPCVControllerV2, change to Deprecated in mainnetAddresses.json, remove from dependencies file and add to deprecatedContractSignoff